src/main/java/athleticli/data/sleep/SleepList.java

```diff
@@ -0,0 +1,57 @@
+package athleticli.data.sleep;
+
+import static athleticli.storage.Config.PATH_SLEEP;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+
+import athleticli.data.Findable;
+import athleticli.data.StorableList;
+
+/**
+ * Represents a list of sleep records.
+ */
+public class SleepList extends StorableList<Sleep> implements Findable {
```

Comment:
Perhaps we can remove this class because most of the methods are not implemented?